modules/exploits/linux/http/trueonline_p660hn_v2_rce.rb

```diff
@@ -98,8 +98,8 @@ def exploit
     @cookie = rand_text_alpha_lower(7)
 
     res = send_request_cgi({
+      'method' => "GET",
```

Comment:
Actually, you can't use `vars_get` here, since there's no key for the query string, just a value.